config/staging/iatlas-api.yaml

```diff
@@ -0,0 +1,20 @@
+template_path: ecs.yaml
+stack_name: iatlas-staging
```

Comment:
please add `stack_tags`, here's an example: https://github.com/Sage-Bionetworks/iAtlas-infra/blob/staging/config/common/iatlasvpc.yaml#L3

Reply:
I've added these and will push them shortly.